libkbfs/cr_actions.go

```diff
@@ -218,29 +218,18 @@ func prependOpsToChain(mostRecent BlockPointer, chains *crChains,
 func crActionConvertSymlink(unmergedMostRecent BlockPointer,
 	mergedMostRecent BlockPointer, unmergedChain *crChain,
 	mergedChains *crChains, fromName string, toName string) error {
-	// If this was turned into a symlink, then we have to simulate
-	// rm/create at the beginning of the mergedOps list.
-	ro, err := newRmOp(fromName, mergedMostRecent)
```

Comment:
Good question: because `makeLocalRenameOpForCopyAction()` prepends a `renameOp` before this, which tells the node cache to move the existing thing out of the way.  I'll updated the comment here to reflect that.